climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1108 @@
+        if total_values is None:
+            total_values = [None] * len(countries)
+        elif len(total_values) != len(countries):
+            raise ValueError("'countries' and 'total_values' must be lists of same length")
+        tag = Tag()
+
+        # litpop_list, a list containing one Exposure instance per country and
+        # None for countries that could not be identified is produced:
+        if admin1_calc: # each admin 1 region initiated seperately,
+                        # with total value share based on subnational GDP share.
+                        # This requires GRP (Gross Regional Product) data in the
+                        # GSDP data folder.
+            litpop_list = [_calc_admin1(country, res_arcsec, exponents, fin_mode,
+                                        total_values[idc], reference_year, gpw_version,
+                                        data_dir, resample_first
+                                        )
+                           for idc, country in enumerate(countries)]
+
+        else: # else, as default, country is initiated as a whole:
+            # loop over countries: litpop is initiated for each individual polygon
+            # within each country and combined at the end.
+            litpop_list = \
+                [self._set_one_country(country,
+                                       res_arcsec=res_arcsec,
+                                       exponents=exponents,
+                                       fin_mode=fin_mode,
+                                       total_value=total_values[idc],
+                                       reference_year=reference_year,
+                                       gpw_version=gpw_version,
+                                       data_dir=data_dir,
+                                       resample_first=resample_first)
+                 for idc, country in enumerate(countries)]
+        # make lists of countries with Exposure initaited and those ignored:
+        countries_in = \
+            [country for i, country in enumerate(countries) if litpop_list[i] is not None]
+        countries_out = \
+            [country for i, country in enumerate(countries) if litpop_list[i] is None]
+        if not countries_in:
+            raise ValueError('No valid country identified in %s, aborting.' % countries)
+        if countries_out:
+            LOGGER.warning('Some countries could not be identified and are ignored: '
+                           '%s. Litpop only initiated for: %s'
+                           % (countries_out, countries_in))
+
+        tag.description = ('LitPop Exposure for %s at %i as, year: %i, financial mode: %s, '
+                           'exp: [%i, %i], admin1_calc: %s'
+                           % (countries_in, res_arcsec, reference_year, fin_mode,
+                              exponents[0], exponents[1], str(admin1_calc)))
+
+        Exposures.__init__(
+            self,
+            data=Exposures.concat(litpop_list).gdf,
```

Comment:
```suggestion
            data=Exposures.concat(countries_in).gdf,
```

Reply:
we actually want to concat litpop_list, not countries_in., countries_in is a list of country names, not of exposures instances.